rb_email_followers/email_followers/extension.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth.models import User
+from P4 import P4
+from reviewboard.extensions.base import Extension
+from reviewboard.extensions.hooks import ReviewRequestPublishedEmailHook
+from reviewboard.reviews.models import Review
+
+def _get_followers(review_request):
+    """Return a list of all `User`s who have configured `p4 reviews` for the files in this review request."""
+    all_reviews = []
```

Comment:
Instead of "reviews" (which I'm guessing you're taking verbatim from the p4 command) Let's use "reviewers".

I'm guessing the p4 command is kinda like asking "who reviews these paths". The answer would be "These are the reviewers" (or more accurately, "these users review the paths", so I guess "users" is OK as well)

Reply:
Yeah I waffled on that. I am fine with changing it. My reasoning was that it was the full output of multiple calls to `p4 reviews`, but that output is in fact just reviewers, so if you as a reader find that more intuitive that's good enough for me.